meshnets/main.py

```diff
@@ -9,32 +9,37 @@
 
 from meshnets.utils import data_loading
 from meshnets.utils import data_processing
+#from meshnets.utils import data_visualization
 
 FLAGS = flags.FLAGS
-
-flags.DEFINE_string('input_object', 'data/motorBike.obj',
+flags.DEFINE_string('input_object', 'data/blob.vtk',
                     'File path of the object to be visualized')
 
 
 def main(_):
 
-    logging.info('Loading the object from %s', FLAGS.input_object)
-    nodes, cells = data_loading.load_mesh_from_obj(FLAGS.input_object,
-                                                   verbose=True)
+    #data_visualization.plot_data(FLAGS.input_object, verbose=True)
+
+    logging.info('Loading the mesh data from %s', FLAGS.input_object)
+    mesh = data_loading.load_edge_mesh_pv(FLAGS.input_object,
+                                          get_pressure=True,
+                                          verbose=False)
+    nodes, edges, pressure = mesh[0], mesh[1], mesh[2]
+    #nodes, edges, pressure = mesh[0], mesh[1], None
 
-    wind_vector = (10, 0, 0)  # dummy wind vector along the X-axis
+    wind_vector = (10, 0, 0)  # wind vector along the X-axis
     # node features for each node are the wind vector
     node_features = np.tile(wind_vector, (len(nodes), 1))
-    null_pressure = np.zeros(shape=(len(nodes), 1))
 
-    logging.info('Building the graph from the mesh')
-    graph = data_processing.triangle_mesh_to_graph(nodes, node_features, cells,
-                                                   null_pressure)
+    logging.info('Building graph from the mesh data')
+    graph = data_processing.edge_mesh_to_graph(nodes, node_features, edges,
+                                               pressure)
 
     logging.info('Node features shape : %s', graph.x.shape)
     logging.info('Edge index shape : %s', graph.edge_index.shape)
     logging.info('Edge features shape : %s', graph.edge_attr.shape)
-    logging.info('Pressure label shape : %s', graph.y.shape)
+    if graph.y is not None:
```

Comment:
"if graph.y:" should be enough, right?

Reply:
I think we should leave it like this. Suppose for example that `y` which I think is the prediction target is `0`. Then the `if` may fail when it shouldn't

Reply:
`if graph.y` is not well defined for arrays so `is not None` works better here.